### DISCUSSION TITLE
extention for accessing powershell/bash in docker desktop

### DISCUSSION THREAD

**Discussion by yogeshdhawale** [docker/extension-ideas#182] [Ideas]
An extension, when opened gives access to local box via shell (ps/bash/gitbash etc).

**Comment by ajeetraina** [comment]
@yogeshdhawale Docker Desktop already have Integrated Terminal for Running Containers starting 4.12.0 release.
Does that solve your problem statement?

<img width="1482" alt="image" src="https://github.com/docker/extension-ideas/assets/313480/691c22f6-bad5-4c12-b97a-20139e43785f">

**Comment by yogeshdhawale** [comment]
I am not referring to access terminal for container. ask is for access to host terminal (mentioned as local box) where I can run commands on host.
